sdk/keyvault/azure-keyvault-secrets/src/main/java/com/azure/security/keyvault/secrets/SecretClient.java

```diff
@@ -518,9 +524,12 @@ public Secret restoreSecret(byte[] backup) {
      * <p>Lists the deleted secrets in the key vault and for each deleted secret prints out its recovery id.</p>
      * {@codesnippet com.azure.security.keyvault.secretclient.listDeletedSecrets}
      *
-     * @return A {@link List} containing all of the {@link DeletedSecret deleted secrets} in the vault.
+     * <p><strong>Samples to iterate DeletedSecret by each page</strong></p>
```

Comment:
the header should be
`<p><strong>Code Samples to iterate keys by page</strong></p>`
Add the description like above + by page.